Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/README.md

```diff
@@ -287,7 +287,6 @@ Create compliance search in the Security & Compliance Center.
 | description | Description of the compliance search. | Optional |
 | allow_not_found_exchange_locations | Whether to include mailboxes other than regular user mailboxes in the compliance search. Default is "false". | Optional |
 | exchange_location | Comma-separated list of mailboxes/distribution groups to include, or you can use the value "All" to include all. | Optional |
-| exchange_location_exclusion | Comma-separated list of mailboxes/distribution groups to exclude when you use the value "All" for the exchange_location parameter. | Optional |
```

Comment:
regenerate the README. 
It can still have it in the README but shown as deprecated. 
Might need to add Deprecated manually